justfile

```diff
@@ -248,6 +246,14 @@ nginx upstream_url='api.openverse.engineering': collectstatic
       -e DJANGO_NGINX_GIT_REVISION="$(git rev-parse HEAD)" \
       openverse-api-nginx:latest
 
+# Launch a psql shell in the web container
+@dbshell:
+    just exec web python manage.py dbshell
+
+# Launch a pgcli shell in the web container (require typing credentials)
+@pgcli db_host="db":
+    just exec web pgcli -h {{ db_host }} openledger deploy
```

Comment:
~~We currently have `set dotenv-load := false` at the top of the file. If we set that to `true` instead, we could pass the `POSTGRES_PASSWORD` in here directly so it doesn't have to be typed manually. Do you think that'd be useful?~~

It looks like there's [no way to override the default `.env` file](https://github.com/casey/just#dotenv-integration) and these are present in `postgres/env.docker` anyway, ah well!